### PR TITLE
Feature/gg 40 research and django testing

### PR DESCRIPTION
**Pull request by orndorffc**
Implements tests for api - authentication, employee, pictures, tempphotos, and models. Implements workflow tests for admin and models.

### CODE REVIEW

**Comment by mraynor** [comment]
There are a couple commits with the same name, can we clean these up/group similar commits?

File: GGProject/tests/test_api_pic.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+            
+            
+    # Test all of the Picture API functionalities
+    def test_pictures(self):
+        
+        print()
+        
+        # Create two employees to add pictures to (emp_ID = 300, 500)
+        self.create_employees()
+        # Add two pictures to employee 300 and one to employee 500
```
Comment (mraynor) [inline GGProject/tests/test_api_pic.py]: It looks like you're adding two to each 300 and 500? (if so, we should update the comment to match)

File: GGProject/workflow/tests/test_models.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    def setUp(self):
+        Employee.objects.create(
+            first_name='first name',
+            last_name='last name',
+            emp_ID = 1,
+            emp_email='email@gmail.com',
+            manager_email='manager@gmail.com',
+            keycode=12345,
+            emp_permissions='0',
+            last_login='2019-01-01'
```
Comment (mraynor) [inline GGProject/workflow/tests/test_models.py]: Can we keep the spacing consistent between `=`?

File: GGProject/workflow/tests/test_models.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        self.assertEqual(expected_object_email, 'email@gmail.com')
+        expected_object_manageremail = f'{employee.manager_email}'
+        self.assertEqual(expected_object_manageremail, 'manager@gmail.com')
+        expected_object_keycode = f'{employee.keycode}'
+        self.assertEqual(expected_object_keycode, '12345')
+        expected_object_permissions = f'{employee.emp_permissions}'
+        self.assertEqual(expected_object_permissions, '0')
+        expected_object_lastlogin = f'{employee.last_login}'
+        self.assertEqual(expected_object_lastlogin, '2019-01-01')
+    #test taht all information shows up on all employees view
```
Comment (mraynor) [inline GGProject/workflow/tests/test_models.py]: Spelling of `that`

File: GGProject/workflow/tests/test_models.py [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+        response = self.client.get(reverse('allpictures'))
+        picture = Picture.objects.get(employee = 1)
+        employee = Employee.objects.get(emp_ID = 1)
+        expected_picture_id = f'{picture.employee}'
+        expected_picture_pic = f'{picture.picture}'
+        expected_picture_name = f'{picture.name}'
+        self.assertEqual(expected_picture_id, employee.__str__())
+        self.assertEqual(expected_picture_pic, 'schematics.png')
+        self.assertEqual(expected_picture_name, 'pic1')
+    #test that all info shows up on single employee page
```
Comment (mraynor) [inline GGProject/workflow/tests/test_models.py:116]: For readability, can we add a new line in between tests?

File: GGProject/tests/test_api_auth.py [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+    # Attempt an unauthenticated requests
+    def test_authenticate(self):
+
+        print()
+        
+        # Attempt an unauthorized request
+        print("SHOULD PRINT 401 ERROR:")
+        self.unauthenticated()
+        print()
```
Comment (mraynor) [inline GGProject/tests/test_api_auth.py]: Will this print anything?

File: README.md [hunk 6]
```diff
@@ -47,4 +70,8 @@ $ cd GGProject && python3 manage.py runserver
 
 **.circleci:** CircleCI directory. Used to automate testing.
 
-**venv:** Virtual environment.
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (mraynor) [inline README.md]: Merge conflict that still needs to be resolved

File: README.md [hunk 7]
```diff
@@ -22,7 +22,30 @@ $ pip3 install -r requirements.txt
 
 ## Running the Application
 ```bash
+<<<<<<< HEAD
```
Comment (mraynor) [inline README.md]: Merge conflict that still needs to be resolved

**Review by lb-cake — DISMISSED** [review]

**Review by lb-cake — COMMENTED** [review]
this PR has erroneous commits. We have decided it would make more sense to create a new branch with a cleaner history clear messages.

**Comment by lb-cake** [comment]
@kemptonk @orndorffc @parksWill have decided to start with a clean slate as the commit history is erroneous and confusing.

**Pull request closed without merging**